apps/web/src/core/routes/AppRoutes.tsx

```diff
@@ -1,38 +1,34 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+
 import type { ReactElement } from "react";
-import { lazy } from "react";
-import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { RouterProvider, Route, createBrowserRouter, createRoutesFromElements, Outlet } from "react-router-dom";
 import { PrivateRouteLogic } from "./logic/PrivateRouteLogic";
 import { PublicRouteLogic } from "./logic/PublicRouteLogic";
 import { PrivateRoutes } from "./PrivateRoutes";
 import { PublicRoutes } from "./PublicRoutes";
 
-const NotFoundPage = lazy(() => import("~/pages/NotFoundPage").then((module) => ({ default: module.NotFoundPage })));
+function Layout(): ReactElement {
+  return <Outlet />;
+}
+
+const router = createBrowserRouter(
```

Comment:
I would like this const to be removed and instead put all the `createBrowserRouter(....)` inside the `return <RouterProvider router={createBrowserRouter(....)} />;` instead. That should also make it possible to remove the eslint-disable 👍